README.md

```diff
@@ -0,0 +1,9 @@
+﻿## Setup
+
+- [ ] Run setup.py file
```

Comment:
`setupy.py` is no longer required because `setup.sh` automatically runs it. We will discuss in on the call